album/undertale-soundtrack.yaml

```diff
@@ -652,8 +652,10 @@ URLs:
 - https://www.youtube.com/watch?v=8CeYNJ-s0Kw
 - https://open.spotify.com/track/2FZ99j224ohiGLavqjijza
 Referenced Tracks:
-- His Theme
 - Once Upon a Time
+Commentary: |-
+    <i>Toby Fox:</i> ([Tumblr](https://undertale.tumblr.com/post/150356633250/before-the-song-undertale-was-completed-it-went))
```

Comment:
Can you just mark this as an excerpt? So just enter it like `([Tumblr](https://...), excerpt)`.

Reply:
Done.